src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -175,10 +176,11 @@ private void createDatePanel() {
                                            I18nUtils.getLocale()
          									+
          									" for Today: ");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) active Embedded Strings : String " for Today: " [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=gyzr%3AGlobalyzerESConcatCheck)